configure

```diff
@@ -1556,7 +1556,7 @@ Optional Packages:
   --with-bonjour          build with Bonjour support
   --with-openssl          build with OpenSSL support
   --without-readline      do not use GNU Readline nor BSD Libedit for editing
-  --with-thrift       do not build thrift
+  --with-thrift       build thrift
```

Comment:
ident issue? run "configure --h", I saw
  --without-readline      do not use GNU Readline nor BSD Libedit for editing
  --with-thrift       do not build thrift
  --without-libyarn       do not build libyarn